src/data/customer/Order.js

```diff
@@ -335,6 +335,10 @@ export const getOrdersAsync = async (filterObject, token) => {
       ? endpoints.API_V3.ORDER_SEARCH
       : endpoints.API_V3.ORDER;
 
+    if (filterObject.keyword) {
+      filterObject.validationStatusId = filterObject.statusIds;
```

Comment:
Looks quite hacky. Something on the backend changes? @leelinde92

Reply:
Yes, back end is not using the same as index. Index uses status_ids while this new end-point uses validation_status_id. I've brought this up in the ticket.